### ISSUE TITLE
[BUG]: Guitar Hero II

### ISSUE THREAD

**Issue by ronnyc7th** [PCSX2/pcsx2#9221]
### Describe the Bug

![Guitar Hero II_SLUS-21447_20230710005815_2](https://github.com/PCSX2/pcsx2/assets/139126773/584e4ae2-b19f-4131-a071-51a19df6df49)

the game Guitar Hero II (US)
the "X" mark is gone or invisible
there should be "X4" under the score points.

doesn't matter what v1.7.xxxx version is (I tried many)
and doesn't matter what the settings is (I also tried many)
the "X" never showed up

tested on laptop ryzen 4600H with gtx1650 (tested on both iGPU and gtx1650)

### Reproduction Steps

just play the game normally

### Expected Behavior

there should be "X4" under the score points.

### PCSX2 Revision

v.1.7.4715

### Operating System

Windows 11

### CPU

amd ryzen 4600H

### GPU

gtx1650

**Comment by refractionpcsx2** [comment]
yes this is a core float rounding bug, which isn't going to be easy to fix, unfortunately.  Some people have bodged this in the past by changing the EE rounding.

**Comment by ronnyc7th** [comment]
gs dump if needed
[snaps.zip](https://github.com/PCSX2/pcsx2/files/12003332/snaps.zip)

I'm not sure, but I think I remember playing guitar hero II on pcsx2 a long time ago when everything runs good,
only little bit lag on view songs

**Comment by seta-san** [comment]
> yes this is a core float rounding bug, which isn't going to be easy to fix, unfortunately. Some people have bodged this in the past by changing the EE rounding.

i can't imagine the value for something that seem extremely static, changing. would it be possible to just hard code in the correct value as a patch?

**Comment by F0bes** [comment]
> > yes this is a core float rounding bug, which isn't going to be easy to fix, unfortunately. Some people have bodged this in the past by changing the EE rounding.
> 
> i can't imagine the value for something that seem extremely static, changing. would it be possible to just hard code in the correct value as a patch?

If you want to find the one magic number that is causing the issue that'd be a start.

**Comment by refractionpcsx2** [comment]
That isn't static, if you don't remember, the whole thing moves when you turn on star power.
